crates/shared/src/bad_token/trace_call.rs

```diff
@@ -73,9 +73,17 @@ impl TraceCallDetector {
         // implementation sending to an address that does not have any balance
         // yet (implicitly 0) causes an allocation.
         let request = self.create_trace_request(token, amount, take_from);
-        let traces = trace_many::trace_many(request, &self.web3)
-            .await
-            .context("trace_many")?;
+        let traces = match trace_many::trace_many(request, &self.web3).await {
+            Ok(result) => result,
+            Err(web3::Error::Transport(e)) => {
```

Comment:
How do we know we want to catch this type of Error? For example, I would rather expect Rpc error with some message like "missing method" or similar?

Reply:
If you received Transport Error instead this might mean the specific node provider has unexpected handling so this code would work only for that fallback provider and not for the rest

Reply:
Very good point. Seems to be an Alchemy specific behavior (Nodereal returns with the response code you expected). Will add this case.